stack/tasks/release.js

```diff
@@ -1,5 +1,6 @@
 const outdent = require('outdent');
 const { exec } = require('child_process');
+
```

Comment:
Is this all of the pr?

Reply:
hehe no, I had to stop and work on other stuff